ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -944,12 +953,76 @@ func isPIInstanceRefreshFunc(client *st.IBMPIInstanceClient, id, instanceReadySt
 	}
 }
 
+func isWaitForPIInstancePlacementGroupAdd(ctx context.Context, client *st.IBMPIPlacementGroupClient, pgID string, id string) (interface{}, error) {
```

Comment:
Could you add constant for the one you just added ? `adding`, `added`, 'deleting', 'deleted'

Reply:
Updated